source/server/server.h

```diff
@@ -406,6 +408,7 @@ class MetricSnapshotImpl : public Stats::MetricSnapshot {
   std::vector<std::reference_wrapper<const Stats::ParentHistogram>> histograms_;
   std::vector<Stats::TextReadoutSharedPtr> snapped_text_readouts_;
   std::vector<std::reference_wrapper<const Stats::TextReadout>> text_readouts_;
+  int64_t snapshot_time_;
```

Comment:
snapshot_time_ms -- and here is a good place for the comment about why we store the integer ms rather than the SystemTime.